Test failure with python 3.13

Unit tests are failing with python 3.13 (found on fedora rawhide):

```
======================================================================
FAIL: testDefaultEnumScope (usb1.testUSB1.USBTransferTests.testDefaultEnumScope)
Enum instances must only affect the scope they are created in.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/xx/pkg/usb1/testUSB1.py", line 289, in testDefaultEnumScope
    self.assertEqual(local_dict.get(ENUM_NAME), ENUM_VALUE)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: None != 42

----------------------------------------------------------------------
Ran 13 tests in 0.079s
```

My understanding of the situation is that the test is not compatible with the new [PEP 667](https://peps.python.org/pep-0667/), specifically the following change:

> For function scopes it will return an instantaneous snapshot of the underlying frame.f_locals.

eg: we do the variable assignment **after** we took a snapshot of the locals.

Downstream fedora bug report: https://bugzilla.redhat.com/show_bug.cgi?id=2280555

--- a/usb1/testUSB1.py
+++ b/usb1/testUSB1.py
@@ -278,15 +278,14 @@ class USBTransferTests(unittest.TestCase):
         """
         ENUM_NAME = 'THE_ANSWER'
         ENUM_VALUE = 42
-        local_dict = locals()
         global_dict = globals()
-        self.assertEqual(local_dict.get(ENUM_NAME), None)
+        self.assertEqual(locals().get(ENUM_NAME), None)
         self.assertEqual(global_dict.get(ENUM_NAME), None)
         self.assertEqual(getattr(libusb1, ENUM_NAME, None), None)
         # pylint: disable=unused-variable
         TEST_ENUM = libusb1.Enum({ENUM_NAME: ENUM_VALUE})
         # pylint: enable=unused-variable
-        self.assertEqual(local_dict.get(ENUM_NAME), ENUM_VALUE)
+        self.assertEqual(locals().get(ENUM_NAME), ENUM_VALUE)
         self.assertEqual(global_dict.get(ENUM_NAME), None)
         self.assertEqual(getattr(libusb1, ENUM_NAME, None), None)
 
@@ -296,16 +295,15 @@ class USBTransferTests(unittest.TestCase):
         """
         ENUM_NAME = 'THE_ANSWER'
         ENUM_VALUE = 42
-        local_dict = locals()
         global_dict = globals()
-        self.assertEqual(local_dict.get(ENUM_NAME), None)
+        self.assertEqual(locals().get(ENUM_NAME), None)
         self.assertEqual(global_dict.get(ENUM_NAME), None)
         self.assertEqual(getattr(libusb1, ENUM_NAME, None), None)
         # pylint: disable=unused-variable
         TEST_ENUM = libusb1.Enum({ENUM_NAME: ENUM_VALUE}, global_dict)
         # pylint: enable=unused-variable
         try:
-            self.assertEqual(local_dict.get(ENUM_NAME), None)
+            self.assertEqual(locals().get(ENUM_NAME), None)
             self.assertEqual(global_dict.get(ENUM_NAME), ENUM_VALUE)
             self.assertEqual(getattr(libusb1, ENUM_NAME, None), None)
         finally:

